configs/sites/noaa-azure/packages.yaml

```diff
@@ -45,7 +45,6 @@ packages:
   berkeley-db:
     externals:
     - spec: berkeley-db@5.3.21
-      prefix: /usr
```

Comment:
How is this working? I thought that spack either needs a prefix or a module to load for external packages?

Reply:
@climbfuji whoops...this was inadvertent. i did this commit in a roundabout way due to github issues on azure.